html/pages/routing/bgp.inc.php

```diff
@@ -240,14 +240,6 @@
             }
         }
 
-        $peerhost = dbFetchRow('SELECT * FROM ipaddr AS A, ports AS I, devices AS D WHERE A.addr = ? AND I.port_id = A.port_id AND D.device_id = I.device_id', array($peer['bgpPeerIdentifier']));
```

Comment:
`$peername` set and never used so removed as query for `$peerhost` is totally wrong (non-existent table).